### PR TITLE
explanate the form to be more obvious for the user

### PR DESCRIPTION
**Pull request by sahdasamier**

### CODE REVIEW

File: src/components/Form/index.tsx [hunk 1]
```diff
@@ -149,6 +150,55 @@ export const Form = ({
...
+      'The information is known from someone else who has firsthand knowledge or experience',
+      'The information is known from a website as a source',
+      'The information is known from a website that has been verified or authenticated.',
+      'The information is known through a verified login or authentication process.',
+      'The information is known through a signed claim or statement',
+      'The information is known through a blockchain system, providing secure and transparent records',
+      'The information is known from a physical document, such as a paper document or certificate',
+      'The information is known through an integrated system or platform'
+    ]
+  }
```
Comment (ArnobChowdhury) [inline src/components/Form/index.tsx:201]: I'd prefer all these strings to be out of the function. If you prefer it in this file, at least take this out of this function. Write this after import but outside the function.

File: src/components/Form/index.tsx [hunk 2]
```diff
@@ -199,9 +257,11 @@ export const Form = ({
               error={Boolean(errors.claim)}
               helperText={errors.claim?.message}
             >
-              {inputOptions.claim.map((i: string) => (
+              {inputOptions.claim.map((i: string, index: number) => (
```
Comment (ArnobChowdhury) [inline src/components/Form/index.tsx:260]: `i` as a variable is widely used to denote `iteration` in programming. Therefore, using `i` to mean something else is very confusing. In this case the ideal name would be `claim` or `claimText` etc.

File: src/components/Form/index.tsx [hunk 3]
```diff
@@ -199,9 +257,11 @@ export const Form = ({
...
                 <MenuItem value={i} key={i}>
-                  {i}
+                  <Tooltip key={i} title={tooltips.claim[index]} placement='right' arrow>
+                    <div> {i}</div>
```
Comment (ArnobChowdhury) [inline src/components/Form/index.tsx:263]: The problem with this is, the tooltip is only over the text and not the whole menu item. To fix this problem, you can use a `Box` component and give it a width and height of 100% of its parent.

**Review by ArnobChowdhury — CHANGES_REQUESTED** [review]
Changes asked for tooltips of `claim` is applicable for `aspect` and `howKnown` as well.

File: src/components/Form/index.tsx [hunk 4]
```diff
@@ -199,114 +257,139 @@ export const Form = ({
...
+                      <TextField
+                        select
+                        label='Aspect'
+                        {...register('aspect')}
+                        sx={{ ml: 1, mr: 1, width: '22ch' }}
+                        margin='dense'
+                        variant='outlined'
+                        fullWidth
+                      >
+                        {inputOptions.aspect.map((i: string, claimText: number) => (
```
Comment (ArnobChowdhury) [inline src/components/Form/index.tsx:348]: Oh, no! not the second argument. Second argument was ok as `index`. I was talking about the first argument, the `i`. Which needs to be `claimText`.

**Review by ArnobChowdhury — CHANGES_REQUESTED** [review]
Check the commentt.

File: src/components/Form/index.tsx [hunk 5]
```diff
@@ -199,114 +257,139 @@ export const Form = ({
...
+                label='How Known'
+                {...register('howKnown')}
+                sx={{ ml: 1, mr: 1, width: '22ch' }}
+                margin='dense'
+                variant='outlined'
+                fullWidth
+              >
+                {inputOptions.howKnown.map((howKnownText: string, index: number) => (
+                  <MenuItem value={howKnownText} key={howKnownText}>
+                    <Tooltip key={howKnownText} title={tooltips.howKnown[index]} placement='right' arrow>
```
Comment (ArnobChowdhury) [inline src/components/Form/index.tsx:280]: Do `Tooltip` need the `key` prop? If not remove it plz.

**Review by ArnobChowdhury — APPROVED** [review]
LGTM. Just left a small comment.

File: src/components/Form/index.tsx (hunk 5 above)
Comment (sahdasamier) [reply src/components/Form/index.tsx]: I removed it, thanks

**Pull request merged**